src/main/resources/application.yml

```diff
@@ -19,6 +19,12 @@ eureka:
       defaultZone: http://localhost:8761/eureka/
   instance:
     hostname: localhost
+keycloak:
+  realm : hackathon-organizer
+  auth-server-url : http://localhost:8080
+  ssl-required : external
+  resource : spring-gateway-client
+  cors : true
 server:
-  port: 0
+  port: 9090
```

Comment:
Proponuje serwisy robic na randomowych portach, brama i tak je znajdzie jak polacza sie do eureki, tu np. na 9090 jest skonfigurowany nasz api-gateway

Reply:
Wiem, narazie mam to tak zostawione bo wygodniej mi sie sprawdza endpointy w postmanie jak przy kazdym uruchomieniu nie dostaje radomowego portu